pkg/sql/pai_submitter.go

```diff
@@ -160,4 +160,31 @@ func (s *paiSubmitter) ExecutePredict(cl *ir.PredictStmt) error {
 	return s.runCommand(code)
 }
 
+func (s *paiSubmitter) ExecuteExplain(cl *ir.ExplainStmt) error {
```

Comment:
I am worrying that the concept of submitter is confusing. In our design doc, submitter means the Python program generated by a codegen.  Here, it seems a kind of mechanism to execute the IR.

Let us follow https://github.com/sql-machine-learning/sqlflow/pull/1703#pullrequestreview-342630202 ASAP to make the followings concepts clear.

1. codegen -- the converter from IR to Python/Couler/etc. programs
1. submitter -- the programs generated from codegen
1. runner -- the mechanism to execute Python/Couler/etc. programs.

Reply:
Thanks for reminding, will do refactor recently.